server/1.3/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM registry.access.redhat.com/ubi8/openjdk-11
+
+LABEL maintainer="Debezium Community"
+
+#
+# Set the version, home directory, and MD5 hash.
+#
+ENV DEBEZIUM_VERSION="1.2.1.Final" \
```

Comment:
Merged this; just one follow-up question: will this get replaced automatically during releases?